### ISSUE TITLE
`try` ... `finally` calls the `finally` block twice

### ISSUE THREAD

**Issue by thennothinghappened** [YoYoGames/GameMaker-Bugs#5654]
### Description
When using the `finally` block on `try` error handling, the block's contents get called twice.

### Steps To Reproduce
**Example Code:**

```gml
var number_of_times_called_finally = 0;

try {
    
    throw "oh no!";
    
} finally {
    show_message($"Called finally {++number_of_times_called_finally} times!");
}

// Shows messages:
// Called finally 1 times!
// Called finally 2 times!
```

### Which version of GameMaker are you reporting this issue for?
IDE v2024.400.0.543 Runtime v2024.400.0.562

### Which operating system(s) are you seeing the problem on?
Mac 13.5

### Which platform(s) are you seeing the problem on?
macOS

[f23e64ad-18ed-42c2-85ed-84f65dc9a4fe](https://api.gamemaker.io/api/github/downloads/f23e64ad-18ed-42c2-85ed-84f65dc9a4fe)

**Comment by rwkay** [comment]
Fixed in 2024.6 - issue was because it was erroneously repeating the final code before the `throw` was executed but both YYC and VM will handle the finally in its own way...

**Comment by mgeddesGM** [comment]
verified on  IDE v2024.600.0.563  Runtime v2024.600.0.580
